include/deal.II/lac/petsc_ts.h

```diff
@@ -93,6 +94,7 @@ namespace PETScWrappers
       const real_type    initial_step_size = 0.0,
       const int          max_steps         = -1,
       const bool         match_step        = false,
+      const bool         restart_if_remesh = false,
```

Comment:
While it is technically an API change, I think we can live with this additional default parameter :smiley: